apps/src/templates/EligibilityChecklist.jsx

```diff
@@ -0,0 +1,198 @@
+/** @file Maker Discount Code Eligibility Checklist */
+import React, {Component, PropTypes} from 'react';
+import i18n from "@cdo/locale";
+import Button from "./Button";
+import SchoolAutocompleteDropdown from './SchoolAutocompleteDropdown';
+import SchoolNotFound from './SchoolNotFound';
+
+import SetupStep, {SUCCEEDED, FAILED, STEP_STATUSES, UNKNOWN} from '../lib/kits/maker/ui/SetupStep';
+
+export default class EligibilityChecklist extends Component {
+  static propTypes = {
+    statusPD: PropTypes.oneOf(STEP_STATUSES).isRequired,
+    statusStudentCount: PropTypes.oneOf(STEP_STATUSES).isRequired,
+  };
+
+  state = {
+    statusYear: UNKNOWN,
+    yearChoice: false, // stores the teaching-year choice until submitted
+    displayDiscountAmount: false,
+    submission: {
+      name: '',
+      email: '',
+      role: '',
+      country: 'United States',
+      hoc: '',
+      nces: '',
+      schoolName: '',
+      schoolCity: '',
+      schoolState: '',
+      schoolZip: '',
+      schoolType: '',
+      afterSchool: '',
+      tenHours: '',
+      twentyHours: '',
+      otherCS: false,
+      followUpFrequency: '',
+      followUpMore: '',
+      acceptedPledge: false
+    },
+    errors: {
+      invalidEmail: false
+    }
+  };
+
+  // Saves the teaching-year choice to trigger next step of actions
+  handleSubmit = () => {
+    this.setState({ statusYear: (this.state.yearChoice ? SUCCEEDED : FAILED )});
+  }
+
+  // Temporarily saves the teaching-year
+  // Param: accept - boolean - true for years to accept, false for deniable answers
+  handleYearChange = (accept) => {
+    this.setState({yearChoice : accept});
+  }
+
+  handleDropdownChange = (field, event) => {
+    this.setState({
+      submission: {
+        ...this.state.submission,
+        [field]: event ? event.value : ''
+      }
+    });
+  }
+
+  handleNotFoundChange = (field, event) => {
+    this.setState({
+      submission: {
+        ...this.state.submission,
+        [field]: event.target.value
+      }
+    }, this.checkShowFollowUp);
+  }
+
+  render() {
+    const {submission, errors} = this.state;
+    return (
+      <div>
+        <h2>
+          {i18n.eligibilityRequirements()}
+        </h2>
+        <div>
+          {i18n.eligibilityExplanation()}
+        </div>
+        <SetupStep
+          stepName={i18n.eligibilityReqPD()}
+          stepStatus={this.props.statusPD}
+        >
+          {i18n.eligibilityReqPDFail()}
+        </SetupStep>
```

Comment:
It feels like a little bit of an abuse of SetupStep to use it for whether teachers meet certain criteria. It might be just that we should rename SetupStep to something that more clearly covers maker setup AND teacher criteria for maker?

Reply:
:+1: Agreed, a rename is appropriate.  Maybe something like `ValidationStep`?

Reply:
This seems like good practice. I think it makes sense to also separate it into a different PR and get that on staging (and merged into this one). Thoughts?

Reply:
I agree that it's fine to do in a separate PR. It's also probably getting this PR merged into staging sooner rather than later (given this is only in storybook so far). I'd actually aim to get this PR merged, then make than rename in a separate PR after this.